doc/index.rst

```diff
@@ -49,3 +124,54 @@ To be completed.
    Topics <topics/index>
    API <Reference_Manual/index>
    FAQ
+
+
+
+
+
+..
+   ********
+   EarthSim
+   ********
+
+   **Lightweight Python tools for environmental simulation**
+
+   EarthSim is a project for developing Python-based workflows for
+   specifying, launching, visualizing, and analyzing environmental
+   simulations, such as those for hydrology modeling. EarthSim is
+   designed to be a lightweight project internally, relying on code and
+   documentation primarily maintained in other, freely available,
+   general-purpose SciPy and `PyViz <http://pyviz.org>`_ projects:
```

Comment:
I assume this is meant for me to update later?  Seems like you could at least replace EarthSim with EarthML and do some minimal updates...